docs/k8s-ibm-public.md

```diff
@@ -0,0 +1,167 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# IBM IKS for OpenWhisk
+
+## Overview
+
+IBM provides both a "Lite" and a "Standard" Kubernetes offering in its
+public cloud Kubernetes service (IKS). These differ in capabilities,
+so are described separately below.
+
+## Initial setup
+
+### Creating the Kubernetes Cluster
+
+Follow IBM's instructions to provision your cluster.
```

Comment:
link? maybe also above for eks?

Reply:
I'm skeptical that links into vendor documentation will be stable; so left it out.